### ISSUE TITLE
More HTTP Statuses

### ISSUE THREAD

**Issue by saiichihashimoto** [feathersjs/feathers-errors#48]
I'd love for this to have 429 and 502. Why not have more of the [default status codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)?

**Comment by daffl** [comment]
You can set any custom status code by throwing an extended [FeathersError](http://docs.feathersjs.com/middleware/error-handling.html):

```js
const FeathersError = require('feathers-errors').FeathersError;

class TooManyRequests extends FeathersError {
  constructor(message, data) {
    super(message, 'TooManyRequests', 429, 'too-many-requests', data);
  }
}

throw new TooManyRequests('You made too many requests');
```

We'd also be happy to accept a PR that adds the missing errors in https://github.com/feathersjs/feathers-errors/blob/master/src/index.js.

**Comment by daffl** [comment]
Closed via #55
